packages/edit-post/src/components/layout/index.js

```diff
@@ -201,11 +196,7 @@ function Layout( { initialPost } ) {
 	}
 
 	const className = clsx( 'edit-post-layout', 'is-mode-' + mode, {
-		'is-sidebar-opened': sidebarIsOpened,
 		'has-metaboxes': hasActiveMetaboxes,
-		'is-distraction-free': isDistractionFree && isWideViewport,
-		'has-block-breadcrumbs':
```

Comment:
This was not used.

Reply:
We need to keep the `has-block-breadcrumbs` class to avoid a regression for [this one](https://github.com/WordPress/gutenberg/issues/53198)